Makefile

```diff
@@ -48,11 +49,16 @@ test: ## Run tests
 
 # build packages
 
-build: clean ## Build an sdist and wheel
+build: ## Build an sdist and wheel
+	$(python) -m pip install --upgrade setuptools wheel
```

Comment:
Independent of this PR's main point, doing 'clean' before attempting to build doesn't work. There is no $(python) to call, because the virtualenv has been cleaned.

In addition, we now update to latest setuptools and wheel before attempting to build wheels.